Excersises/exercise6.js

```diff
@@ -0,0 +1,18 @@
+/* Write a JavaScript function to check if a given string is a palindrome (reads the same forwards and backwards).*/
+
+const checkPalindrome = (str) => {
+  let righToleft = "";
+  let leftToRight = str;
+
+  for (let i = str.length; i >= 0; i--) {
+    righToleft += str.charAt(i);
```

Comment:
please access string character like this : str[i]

Reply:
Not working!